docs/CreateKoblenzHash.md

```diff
@@ -0,0 +1,62 @@
+# Creating Hashes for Koblenz Pass Data
+
+## Steps
+
+The example data is
+
+| Parameter    | Value         |
+|--------------|---------------|
+| Name         | Karla Koblenz |
+| Geburtstag   | 10.06.2003    |
+| Aktenzeichen | 123K          |
+
+
+### 1. Collect all data and merge it into an object
+```agsl
+//Example:
+{
+    full_name: "Karla Koblenz"
+    birthday: 12213
+    referenceNumber: "123K"
+}
+```
+
+- The full_name must be `FirstnameSpaceLastname`. Every char must exactly match the user input, as otherwise there is not possibility to match the data Koblenz transfers with the input the user makes.
+e.g. `Karla Koblenz` will match neither with `Karla Lisa Koblenz` nor with `Karlá Koblenz`.
```

Comment:
So improve matching we could create multiple hashes that e.g. hash the initials. I don't know the usecase but this might be an option if issues arise.